### ISSUE TITLE
test_sirepo_flyer_vcr should not depend on ordering of results

### ISSUE THREAD

**Issue by jklynch** [NSLS-II/sirepo-bluesky#19]
`test_sirepo_flyer_vcr` asserts a particular ordering of simulation results. This results in test failures even when the correct results have been found. For example:
```
>       assert actual_means == db_means, "fly scan means do not match actual means"
E       AssertionError: fly scan means do not match actual means
E       assert [36779651609602.38,\n 99449330615601.89,\n 149289119385413.34,\n 223428480785808.78,\n 388594677365777.9] == [149289119385413.34,\n 36779651609602.38,\n 388594677365777.9,\n 99449330615601.89,\n 223428480785808.78]
E         At index 0 diff: 36779651609602.38 != 149289119385413.34
E         Full diff:
E           [
E         +  36779651609602.38,
E         +  99449330615601.89,
E            149289119385413.34,
E         -  36779651609602.38,
E         +  223428480785808.78,
E            388594677365777.9,
E         -  99449330615601.89,
E         -  223428480785808.78,
E           ]
sirepo_bluesky/tests/test_sirepo_flyer.py:70: AssertionError
```

**Comment by mrakitin** [comment]
GitHub's "magic" keyword `Closes #<>` only worked for the first issue (#7). For it to work for all listed issues it has to use `Closes` for every issue we want to close.

Closing it here manually - resolved via #15.

**Comment by mrakitin** [comment]
Documentation on that: https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword.
